tests/trainers/test_byol.py

```diff
@@ -60,20 +49,20 @@ def test_custom_augment_fn(self) -> None:
 
 
 class TestBYOLTask:
-    @pytest.mark.parametrize(
-        "name,classname",
-        [
-            ("chesapeake_cvpr_7", ChesapeakeCVPRDataModule),
-            ("chesapeake_cvpr_prior", ChesapeakeCVPRDataModule),
```

Comment:
We could test any of our 50+ supervised datasets, but I think it makes more sense to test our unsupervised datasets. Right now, we only have SeCo and BYOL, but I'm working on adding SSL4EO and SimCLR/MoCo which will also be tested in a similar fashion.